RunningOrder/Sprint/NewSprintView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  NewSprintView.swift
+//  RunningOrder
+//
+//  Created by Clément Nonn on 07/07/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+
+struct NewSprintView: View {
+    @State private var name: String = ""
+
+    @Binding var createdSprint: Sprint?
+    @Environment(\.presentationMode) var presentationMode
+
+    var body: some View {
+        VStack {
+            TextField("Sprint Name", text: $name, onEditingChanged: { _ in }, onCommit: createSprint)
```

Comment:
I think `onEditingChanged` can be omitted :)

Reply:
Nope the parameter is required xD I wanted to remove it but it is not possible